libr/asm/d/v850.sdb.txt

```diff
@@ -19,27 +22,48 @@ bnl=branch if not carry/less than
 bnv=branch if not overflow
 bp=branch if positive
 br=branch always
+bsh=byte swap halfword
+bsw=byte swap word
 bv=branch if overflow
+callt=call with table lookup
+caxi=compare and exchange for interlock
 caxi=compare and exchange interlocked
 cli=clear interrupt disable flag
-cmp=compare
+clr1=clear bit
+cmov=conditional move
+cmp=compare two registers and update flags
 cmpf.s=compare floating short
+ctret=return from callt
 cvt.sw=convert floating short to word
 cvt.ws=convert word to floating short
+di=disable interrupts
+dispose=function dispose
 div=divide signed
 divf.s=divide floating short
+divh=divide half word
+divq=divide quad word
 divu=divide unsigned
+ei=enable interrupts
+eiret=return from trap or interrupt
+feret=return from trap or interrupt
+fetrap=software trap
 halt=halt cpu
+hsh=halfword swap half word
+hsw=halfword swap word
 in.b=input byte
 in.h=input halfword
 in.w=input word
 jal=jump and link
-jmp=jump register
-jr=jump relative
-ld.b=load byte
+jarl=jump to subroutine and save old pc in given register
+jmp=unconditional jump
```

Comment:
jump register

Reply:
it's register relative, and can also take an imm for disp. but i think the description is pretty clear without so much details. ill change the text with your suggestion

![Screenshot 2020-10-08 at 20 24 34](https://user-images.githubusercontent.com/6431515/95498566-4d3c0e00-09a4-11eb-90a2-1dd44b281a59.png)

Reply:
yes, but why so many words? `jump register` is quite clear definition of all that jargon.